addons/medical/CfgVehicles.hpp

```diff
@@ -86,6 +86,18 @@ class CfgVehicles {
                 typeName = "NUMBER";
                 defaultValue = 1;
             };
+            class maxUnconsciousTime {
+                displayName = "Base Maximum Unconscious Time";
+                description = "The base amount of time that a unit will spend unconscious before waking up without medical intervention."; // TODO: Stringtable
+                typeName = "NUMBER";
+                defaultValue = 10; // testing value: 10
```

Comment:
Considering default in ACE_Settings is -1, i.e. disabled, shouldn't the module default option be something different, like 300 as an example?